zerver/actions/users.py

```diff
@@ -604,6 +606,12 @@ def get_service_dicts_for_bot(user_profile_id: int) -> list[dict[str, Any]]:
         # A ConfigError just means that there are no config entries for user_profile.
         except ConfigError:
             return []
+    elif user_profile.bot_type == UserProfile.INCOMING_WEBHOOK_BOT:
+        return [
+            {
+                "integration_name": get_bot_config(user_profile).get("integration_name"),
```

Comment:
We can return the data structure similar to embedded bot's just above this condition to make things a bit more consistent.
```
try:
    return [
        {
            "config_data": get_bot_config(user_profile),
        }
    ]
except ConfigError:
    return []
```
Don't forget to except `ConfigError`, since the "integration_name" field is not mandatory and can be left empty

Reply:
Changes made